lib/app/pages/splash.dart

```diff
@@ -45,6 +45,10 @@ class _SplashPageState extends State<SplashPage> {
         await context.read<ThemeCubit>().getCurrentTheme();
       },
     );
+    Future.delayed(
```

Comment:
has no effect during my tests

Reply:
thank you for catching...
introduced KeyStatus.resetKey to handle flow more effectively